packages/babel-preset-app/src/index.js

```diff
@@ -48,6 +68,12 @@ module.exports = (context, options = {}) => {
     absoluteRuntime
   } = options
 
+  let { corejs = { version: 2 } } = options
+
+  if (typeof corejs === 'number') {
```

Comment:
I think `version` type can be also a `string` for minor versions no?

Reply:
Unfortunately it only supports number https://github.com/babel/babel/blob/master/packages/babel-plugin-transform-runtime/src/index.js#L57

Reply:
`Number('2.3.1')` is `NaN` but i think still makes sense to allow passing values like `'2'`.

Reply:
Right, actually babel only supports `2, 3, '2', '3', false`